localplanning.services/src/pages/search.astro

```diff
@@ -0,0 +1,15 @@
+---
+import Layout from "../layouts/Layout.astro";
+import { ReactTest } from "../components/ReactTest";
+---
+
+<Layout>
+  <ReactTest client:load />
+  <hr/>
+  Some example URLs - 
+  <ul>
+    <li><a href="/buckinghamshire">Bucks</a></li>
+    <li><a href="/camden">Camden</a></li>
+    <li><a href="/lambeth">Lambeth</a></li>
```

Comment:
To match a postcode to a PlanX team I think we'll need to - 
 - take the lat/lng (or x&y) returned by the OS Places API
 - query the planning data API to get the LPA
 - Match to a PlanX team
 
 We can wrap this up as an endpoint in the PlanX REST API.